src/Stratis.Bitcoin.Features.Dns.Tests/GivenADnsFeature.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Stratis.Bitcoin.P2P;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.Dns.Tests
+{
+    /// <summary>
+    /// Tests for the <see cref="DnsFeature"/> class.
+    /// </summary>
+    public class GivenADnsFeature
+    {
+        [Fact]
+        [Trait("UnitTest", "UnitTest")]
```

Comment:
Would naming the trait `[Trait("Dns", "UnitTest")]` be more useful?

Reply:
Good point, makes it easier then to just run the Dns tests.